app/gofundme/page.tsx

```diff
@@ -0,0 +1,174 @@
+import Link from "next/link"
+import { Button } from "@/components/ui/button"
+import Image from "next/image"
+
+import mangrove from "@/assets/fundraising/mangrove.jpg"
+import mangrove2 from "@/assets/fundraising/mangrove222.png"
+import map from "@/assets/fundraising/map.png"
+import logo from "@/assets/logo-main-pr.png"
+
+export default function GoFundMe() {
+  return (
+    <div className="flex flex-col min-h-[100dvh]">
+      <main className="flex-1">
+        <section className="w-full py-12 md:py-24 lg:py-32 bg-gray-100 dark:bg-gray-800">
+          <div className="container px-4 md:px-6">
+            <div className="grid gap-6 lg:grid-cols-[1fr_550px] lg:gap-12 xl:grid-cols-[1fr_650px]">
+              <div className="space-y-4">
+                <h1 className="text-3xl font-bold tracking-tighter sm:text-5xl xl:text-6xl/none" style={{ color: '#0F4C32' }}>
```

Comment:
styling nya pake tailwind bisa diganti jadi gini instead of nambahin additional styles, biar lebi rapih. Tapi gapapaa!

```suggestion
                <h1 className="text-3xl font-bold tracking-tighter sm:text-5xl xl:text-6xl/none text-[#0F4C32]" >
```

instead of pake style, biar lebih rapih codenya